### ISSUE TITLE
I can't connect to unprotected networks with the configurator 

### ISSUE THREAD

**Issue by lekkas** [resin-os/resin-device-toolbox#8]
Connects to https://github.com/resin-io-playground/resin-toolbox/issues/47

**Comment by jviotti** [comment]
Can you elaborate a bit more on this? What does "unprotected" network means? Omitting a network key yields an error? If so, what error?

**Comment by shaunmulligan** [comment]
@jviotti he means "unprotected" in the sense that the wifi has no password. Now the issue here is that it's not as easy as setting an empty string for `psk`, for NM connection files, you need to remove the enire [`wifi-security`] section and a few other things, so we need to think about a good way to do this :(

**Comment by jviotti** [comment]
I see. I guess the only option is to introduce a "checkbox" for "Does your network needs a password?", and conditionally apply the whole thing like we were doing for ethernet and wifi in the ResinOS v1 reconfix example.

**Comment by shaunmulligan** [comment]
@jviotti yeah, that sounds like a good approach to me. In the CLI it does require the user to go through another step, but I dont see an easy way around that right now.
